web.rb

```diff
@@ -110,9 +110,25 @@ def authenticate!
     else
       begin
         @customer = create_customer()
+
+        payment_methods = []
+
+        ['4000000000003220', '4000000000003238', '4000000000003246', '4000000000003253', '4242424242424242'].each { |cc_number|
```

Comment:
Let's create and attach in the same block. Can you also add a comment pointing to https://stripe.com/docs/payments/3d-secure#three-ds-cards?